docker/Dockerfile.centos

```diff
@@ -25,3 +25,6 @@ RUN curl -sL https://github.com/shyiko/jabba/raw/master/install.sh | JABBA_COMMA
 
 RUN echo 'export JAVA_HOME="/jdk"' >> ~/.bashrc
 RUN echo 'PATH=/jdk/bin:$PATH' >> ~/.bashrc
+
+# when the JDK is GraalVM install native-image
+RUN test -O /jdk/bin/gu && /jdk/bin/gu install native-image
```

Comment:
@vjovanov you need to check this as otherwise it will fail to build the docker image when we not use GraalVM.

I just verified that this works:

```
+RUN if [ -O /jdk/bin/gu ]; then /jdk/bin/gu install native-image; else echo "Not GraalVM, skip installation of native-image" ; fi
```

Once this change was done I can merge it. Good work!

Reply:
Thanks, done.